tareas/AndresCarlos/Tarea01.jl

```diff
@@ -0,0 +1,377 @@
+	end
+	
+	
+	function Intervalo()     #Intervalo vacio
+		return ()      #Regresamos un arreglo pues vacio :v
+	end
+	
+end
+
+# ╔═╡ b6be6e7a-3ae9-48ba-abf6-bd04e889164b
+Intervalo()
+
+# ╔═╡ a2d08992-2591-4f1d-a8e0-43076aa9d5e2
+function ∪(a::Intervalo, b::Intervalo) #Definimos la operación union#
+	CotI=min(a.infimo, b.infimo) #Definimos la cota minima de los dos intervalos
+	CotS=max(a.supremo, b.supremo) #Definimos la cota maxima de los dos intervalos
+	return Intervalo(CotI, CotS) #Regresamos la función intervalo con esas cotas
+end
+
+# ╔═╡ 477da29e-8326-48aa-b823-01cc86c4d4b3
+begin
+	I1=Intervalo(3,4)
+    I2=Intervalo(5,9)
+	I3=Intervalo(1,10)
+	I4=Intervalo(9,16)
+	zero=Intervalo(0,0)
+end
+
+# ╔═╡ d1affbbd-b74e-4983-a6a2-92a688ab6c19
+∪(I1,I2)
+
+# ╔═╡ 5b13850e-1e68-461b-a41c-2cd695502716
+function ∩(a::Intervalo, b::Intervalo) #definimos la función de intersección
+	if a.supremo < b.infimo    #Vemos que el intervalo a y b son ajenos con a.sup < b.inf
+		return Intervalo() #de ser el caso nos arroja el vacio
+	elseif b.supremo < a.infimo  #Vemos que sean ajenos para el otro caso que a este a la 
+		                  #derecha de b
+		return Intervalo() #De ser el caso nos arroja el intervalo vacio
+	elseif b.supremo < a.supremo && b.supremo > a.infimo  #vemos un primer caso de la intersección
+		CotMin= max(a.infimo, b.infimo)  #como b esta atrapado en a, b.sup < a.sup, la cota
+		return Intervalo(CotMin, b.supremo) #minima estara denotada por la maxima de los 
+		                                #inf
+	elseif a.supremo < b.supremo && a.supremo > b.infimo #El otro caso donde a.sup < b.sup
+		CotMin= max(a.infimo, b.infimo)   #Vemos la cota inf adecuada para la intersección
+		return Intervalo(CotMin, a.supremo) #Nos regresa la intersección
+		
+	end
+end 
+
+# ╔═╡ 6a5926ee-e419-4ea4-9ed2-fe1dd110de89
+begin 
+	
+	import Base: == #No se lo entiendo del todo pero me pedia importar :V
+	
+	function ==(a::Intervalo, b::Intervalo) #Definimos la función de igualdad
+		
+		return a.infimo == b.infimo && a.supremo == b.supremo
+		
+	end
+	
+end
+		
+
+# ╔═╡ 94aa2723-4d33-4a1d-b9f0-eae10c02f157
+I1 == I2
+
+# ╔═╡ 85117a02-e8fb-40e9-b9dc-43e1b599179a
+begin 
+	
+	import Base. ∈
+	
+	function ∈(x::Real, j::Intervalo) #Definimos el operador logico ∈ ¨creo¨
+		
+		return j.infimo ≤ x ≤ j.supremo 
+		
+	end
+	
+end
+
+# ╔═╡ cb54faf9-cd66-4459-b3f9-9917b6d5df7b
+7 ∈ I2
+
+# ╔═╡ 38542482-adfd-45d9-8190-c823e63740cf
+function ⪽(a::Intervalo, b::Intervalo) #Definimos el operador logico ⪽ ¨creo, no se 
+	                                   #decirlo mejor¨
+	return a.infimo > b.infimo && a.supremo < b.supremo
+	
+end
+
+# ╔═╡ 1eac0dc7-89db-42fe-bff5-d5db0b317383
+I1 ⪽ I3
+
+# ╔═╡ 9b51c7e4-7c48-4f46-8e52-4f8d4fbd6cbe
+function ⊃(a::Intervalo, b::Intervalo) #Definimos el operador logico ⊃ ¨creo, no se 
+	                                   #decirlo mejor¨ 
+	return b.infimo > a.infimo && b.supremo < a.supremo 
+	
+end
+
```

Comment:
```suggestion
import Base: ⊆
```
Agrega esto para su correcto funcionamiento